pkg/controller/controller_binding.go

```diff
@@ -58,11 +63,17 @@ func (c *controller) reconcileServiceBindingKey(key string) error {
 	}
 	binding, err := c.bindingLister.ServiceBindings(namespace).Get(name)
 	if apierrors.IsNotFound(err) {
-		glog.Infof("Not doing work for ServiceBinding %v because it has been deleted", key)
+		glog.Infof(
+			`%s "%s/%s": Not doing work for ServiceBinding %v because it has been deleted`,
+			typez, namespace, name, key,
+		)
 		return nil
 	}
 	if err != nil {
-		glog.Infof("Unable to retrieve ServiceBinding %v from store: %v", key, err)
+		glog.Infof(
+			`%s "%s/%s": Unable to retrieve ServiceBinding %v from store: %v`,
```

Comment:
Message should be "...: Unable to retrieve from store"

Reply:
I am keeping the error